rebar.config

```diff
@@ -0,0 +1,32 @@
+
+{erl_opts, [{parse_transform, lager_transform}]}.
+
+{deps, [
+        {lager, {git, "https://github.com/erlang-lager/lager.git",
+                 {ref, "69b4ada"}}}, % tag: 3.6.7
+
+        %% Cuckoo prebuilt CUDA binaries.
+        {aecuckooprebuilt,
+         {aecuckooprebuilt_app_with_priv_from_git, {git, "https://github.com/aeternity/cuckoo-prebuilt.git",
+                                                    {ref, "90afb699dc9cc41d033a7c8551179d32b3bd569d"}}}},
+
+        %% Cuckoo CPU miners (not prebuilt).
+        {aecuckoo, {git, "https://github.com/aeternity/aecuckoo.git",
+                    {ref, "fa3d13e"}}},
+
+        %% This is used just in one place, just to get blake2b_256 hash.
```

Comment:
Not sure if the hash is performance critical, there is a native Erlang implementation that we use in `aesophia` but possibly better to use a NIF here.